### PR TITLE
Added release notes for Azure Spring Boot

### PR DESCRIPTION
**Pull request by yiliuTo**

### CODE REVIEW

File: releases/2020-08/java.md [hunk 1]
```diff
@@ -264,6 +265,25 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - MessageDetailCount is flattened and removed in QueueRuntimeInfo, TopicRuntimeInfo, and SubscriptionRuntimeInfo.
 >>>>>>> 909ac89a4a5bac3741c7f777d8dc3a6370fa1343
 
+### Azure Spring Boot ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot/CHANGELOG.md#233-2020-08-07))
```
Comment (samvaity) [inline releases/2020-08/java.md:268]: Ideally the changelog date should have been of today.

File: releases/2020-08/java.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ The Azure SDK team is pleased to announce our August 2020 client library release
 - Azure Core Http Netty
 - Azure Core Http OkHttp
 - Azure-Cosmos
+- Azure Spring Boot
```
Comment (samvaity) [inline releases/2020-08/java.md:22]: Do we just need to mention about this package?
What about the other starter packages, they don't need to be mentioned?

File: releases/2020-08/java.md [hunk 3]
```diff
@@ -264,6 +265,25 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - MessageDetailCount is flattened and removed in QueueRuntimeInfo, TopicRuntimeInfo, and SubscriptionRuntimeInfo.
 >>>>>>> 909ac89a4a5bac3741c7f777d8dc3a6370fa1343
 
+### Azure Spring Boot ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot/CHANGELOG.md#233-2020-08-07))
+
+#### New Features
```
Comment (samvaity) [inline releases/2020-08/java.md:270]: Also, for these headings follow from one of these https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

File: releases/2020-08/java.md (hunk 2 above)
Comment (yiliuTo) [reply releases/2020-08/java.md:24]: Hi,  Azure Spring Boot here doesn't present the package name but a total description of the spring service, besides given that users don't use the package azure-spring-boot directly, so I think it won't bring ambiguity here.

**Comment by yiliuTo** [comment]
Hi @samvaity  In order to let users know about the latest version of Spring modules, I have added the versions in "Installation Instructions" section. While considering that Spring packages are not suitable for placing together with other services because when users import Spring, they don't need to import SDK packages, thus I put Spring packages independently from others. Do you thinks it can work?

File: releases/2020-08/java.md [hunk 4]
```diff
@@ -126,6 +127,45 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
 </dependency>
 ```
 
+To use Azure Spring Boot starters, refer to the Maven dependency information below, which may be copied into your projects Maven `pom.xml` file as appropriate.
```
Comment (samvaity) [inline releases/2020-08/java.md:130]: @czubair Are we fine with adding this section for Spring releases or do we have any other guidance for this ?

Comment (czubair) [reply releases/2020-08/java.md:130]: This is fine.

**Pull request merged**